### DISCUSSION TITLE
SSL Certificate

### DISCUSSION THREAD

**Discussion by Lewinta** [frappe/gameplan#330] [Q&A]
I did the installation and I can see that the site is up using Curl, however I cannot access because I have SSL on my site and nginx is set to forward any traffic from 80 to 443.
From HTTP to https and I cannot access

**Comment by jabaharjagadishbaral** [comment]
If you're running Gameplan on a Frappe Bench installation with your own domain, you can use Bench's Let's Encrypt setup for SSL.

For example:

```bash
sudo bench setup lets-encrypt your-domain.com
```

After the certificate is configured, Bench can generate the NGINX configuration and reload NGINX. The current Bench documentation also provides commands for configuring an SSL certificate and private key manually.

Make sure your domain's DNS is pointing to the server before requesting the certificate, and that ports 80/443 are accessible.

If you're using Docker instead of a normal Bench installation, the SSL setup is different, so the deployment method matters.
